src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -121,6 +148,26 @@ export class QuickBar extends LitElement {
     `;
   });
 
+  private _handleInputKeyDown(ev: KeyboardEvent) {
+    if (ev.code === "Enter") {
+      if (this._commandMode) {
+        this._runCommandAndCloseDialog(this._topCommandResult);
+      } else {
+        this._launchMoreInfoDialog(this._topEntityIdResult);
```

Comment:
Wouldn't it be easier to just do:
```suggestion
        this._firstListItem.click();
```

Reply:
Done